README.md

```diff
@@ -61,6 +61,7 @@ releases of the toolkit.
     * Python 3.6.2, along with a set of additional Python packages, is required to run some tools and workflows.
       See [Python Dependencies](#python) for more information.
     * R 3.2.5 (needed for producing plots in certain tools)
+    * The GCC OpenMP library, `libgomp1`, is needed in order to run the accelerated PairHMM implementation in `HaplotypeCaller` / `Mutect2`. This library is installed automatically when you install `gcc`. If it's not installed, GATK will fall back to using the much slower Java PairHMM implementation.
```

Comment:
Should we mention this is unecessary on mac because it doesn't run OMP?

Reply:
Hmm....now I'm wondering why it can't fall back gracefully on Linux, and instead reports that AVX is not available. Do you understand the reason @lbergelson ?

Reply:
Not clear why, it seems like a bug somewhere.  AVX is available, there's some issue with OMP, unclear why it' not included / identified earlier.

Reply:
It might actually be an issue with the fallback logic in the GATK PairHMM class -- we should check on that before merging this PR.